django3_codemods/config_tools/tox_modernizer.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import re
+from configparser import ConfigParser, NoSectionError
+
+TOX_SECTION = "tox"
+ENV_SECTION = "envlist"
+TEST_ENV_SECTION = "testenv"
+TEST_ENV_DEPS = "deps"
+PY_38 = "py{38}"
+DJANGO_22 = "django{22}"
+
+DJANGO_22_DEPENDENCY = " django22: Django>=2.2,<2.3\n"
```

Comment:
Why just one leading space in the string?

Reply:
it should lead with a tab (\t)? if we're replacing the whole line

Reply:
I think most of our tox.ini files use four space indentation, not tabs.

Reply:
@jmbowman I've ran the script on some repos for testing and having four-space indentation is inconsistent, should we go ahead and add four leading spaces? considering four-space indentation to be the standard.